Stackdriver-Webhook/README.md

```diff
@@ -0,0 +1,184 @@
+
+
+# Catchpoint Google Stackdriver
+
+  
+
+####
+
+## **Introduction**
+
+  
+
+Cloud Monitoring provides visibility into the performance, uptime, and overall health of applications. It collects metrics, events, and metadata from Google Cloud, Amazon Web Services, hosted uptime probes, application instrumentation, and a variety of common application components including Cassandra, Nginx, Apache Web Server, Elasticsearch, and many others. Operations ingests that data and generates insights via dashboards, charts, and alerts. Cloud Monitoring alerting helps you collaborate by integrating with Slack, PagerDuty, and more.
+
+  
+
+####
+
+  
+  
+
+We will be looking at ingesting data from Catchpoint into Cloud Monitoring.
+
+[https://cloud.google.com/monitoring](https://cloud.google.com/monitoring)
+
+  
+
+
+
+## **Installation &amp; Configuration**
```

Comment:
We can remove the '**' here and in later heading as well.

Reply:
changed on the latest commits